lib/inspec/resources/service.rb

```diff
@@ -776,7 +816,14 @@ class BSDService < Service
     EXAMPLE
 
     def select_service_mgmt
-      BSDInit.new(inspec, service_ctl)
+      os = inspec.os
+      version = os[:release].to_f
+
+      if version >= 10
+        FreeBSD10Init.new(inspec, service_ctl)
+      else
+        BSDInit.new(inspec, service_ctl)
+      end
```

Comment:
I'm not 100% sure if it's the right approach to add an automatic selection inside the `BSDService` type, rather than add a second service type? The main thing that led me to this approach was trying to decide, if it was two service types, which one (FreeBSD10+ or the old one) would map to the existing `bsd_service` name. Most people presumably are on 10+ by now, but it would be backwards compatibility-breaking to default to a method that doesn't work on 9. In the end it seemed like this was the win/win approach, but I'm not married to it.